client/js/navigation/config.js

```diff
@@ -23,7 +22,26 @@ export const sharedNavigationOptions = navigation => ({
   },
   headerLeft: (
     <TouchableOpacity onPress={() => navigation.toggleDrawer()}>
-      <Icon name="md-menu" size={25} color="white" style={{ margin: 15 }} />
+      <Image
+        source={require("../assets/images/hamburgermenu.png")}
```

Comment:
Use react-native-vector-icons instead of an image.

Reply:
Done.